### ISSUE TITLE
Add support for R11F_G11F_B10F, and/or fix IBL instructions in Sandcastle

### ISSUE THREAD

**Issue by emackey** [CesiumGS/cesium#9088]
In the [Image-Based Lighting Sandcastle Demo](https://sandcastle.cesium.com/?src=Image-Based%20Lighting.html&label=All), there are comments recommending how to create a `.ktx` file for use as an HDRI lighting environment.  The instructions recommend downloading [Google Filament](https://github.com/google/filament/releases) and using their cmgen tool (from the bin folder).

So I tried that with Filament 1.8.1 on a free HDRI called Symmetrical Garden (from HDRI Haven), and the result is this KTX file:

[symmetrical_garden_1k_ibl.zip](https://github.com/CesiumGS/cesium/files/5053082/symmetrical_garden_1k_ibl.zip)

The KTX file came out of Filament with a pixel format of `R11F_G11F_B10F` (`35898`), which is not supported in Cesium.

Can support for this pixel format be added?  (I tried naively adding the format to the supported list, but clearly it takes a little more than that to get it working).

**Comment by emackey** [comment]
Worth noting that the old cmgen, at the time this demo was first published, had this note in the command-line help message:

>        KTX files are always encoded with 4-channel RGBM data

But the latest cmgen has this note instead:

>        KTX files are always encoded with 3-channel RGB_10_11_11_REV data

**Comment by lilleyse** [comment]
@YoussefV you ran into something similar right? I know there were a few open threads about Filament, the KTX1->KTX2 converter, and CesiumJS. Where did we settle on those?

CC https://github.com/CesiumGS/cesium/pull/9040

**Comment by YoussefV** [comment]
Yep, so for `R11F_G11F_B10F` pixel formats, you can use the `WebGLConstants.R11F_G11F_B10F` for the format of the image. The IBL sandcastle instructions didn't work for me either when i tried converting the kiara environment map to `.ktx2`, so I had to basically rewrite the original `.ktx` binary.
